doc/source/administrator/authentication.md

```diff
@@ -0,0 +1,427 @@
+(authentication)=
+
+# Authentication and authorization
+
+Authentication is about identity, while _authorization_ is about access rights.
+In this section you will learn how to configure both. For example, you can
+configure to authenticate GitHub users and authorize those part of a specific
+GitHub organization.
+
+Before configuring this, you should have [setup HTTPS](https).
+
+## Useful understanding
+
+### Authenticator classes
+
+As JupyterHub can't know how to authenticate all kinds of users, it expose
+configuration to extend its capabilities to rely on a dedicated [_authenticator
+class_](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html).
+Several such classes are already made available through [installed Python
+packages](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/images/hub/requirements.txt).
+
+### The Authenticator base class
+
+JupyterHub provides a base class,
+[`Authenticator`](https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/auth.py),
+that all other authenticator classes are supposed to derive from. By configuring
+this base class, we influence the behavior of the derived class as well.
+
+### The configuration system
+
+We configure the JupyterHub to use our chosen authenticator class and the
+authenticator class in question through this Helm chart's
+[`hub.config`](schema_hub.config) configuration.
+
+## Configuring authorization
+
+Some authenticator classes contain dedicated authorization logic, but that
+doesn't stop you from using the common base class authorization logic.
+
+```yaml
+hub:
+  config:
+    Authenticator:
+      admin_users:
+        - user1
+        - user2
+      allowed_users:
+        - user3
+        - user4
+    # ...
+    DummyAuthenticator:
+      password: a-shared-secret-password
+    JupyterHub:
+      authenticator_class: dummyauthenticator.DummyAuthenticator
+```
+
+```{note}
+Please refer to [JupyterHub's own documentation](https://jupyterhub.readthedocs.io/en/latest/getting-started/authenticators-users-basics.html) for more details.
+```
+
+## Configuring authenticator classes
+
+### OAuth2 based authentication
+
+JupyterHub's
+[oauthenticator](https://github.com/jupyterhub/oauthenticator) has
+support for enabling your users to authenticate via a third-party OAuth
+provider, including GitHub, Google, and CILogon.
+
+Follow the service-specific instructions linked on the [oauthenticator
+repository](https://github.com/jupyterhub/oauthenticator) to generate
+your JupyterHub instance's OAuth2 client ID and client secret. Then
+declare the values in the helm chart (`config.yaml`).
+
+Here are example configurations for common authentication services. Note
+that in each case, you need to get the authentication credential
+information before you can configure the helm chart for authentication.
```

Comment:
```suggestion
that in each case, you need to get the client ID and secret
before you can configure the helm chart for authentication.
```
I think repeating this is clearer than introducing a new term "authentication credential information"